.github/workflows/test-hyperlinks.yml

```diff
@@ -226,6 +226,11 @@ jobs:
               if [[ ${REFERENCE:0:1} == '#' ]]; then 
                 LINK_TEXT=$( cat $FILE | grep  -oE "\[.*\]\(${REFERENCE}\)" | sed 's/\[//' | cut -d ']' -f1 )
                 IS_PRESENT=$(cat $FILE | grep -oE "# ${LINK_TEXT}" | wc -w)
+                if [[ $IS_PRESENT -eq 0 ]]
```

Comment:
Should probably keep this as if [[ test ]]; then  like the other if statements in the file for readability.  Otherwise everything looks fine.